src/py/flwr/cli/new/new.py

```diff
@@ -41,6 +41,16 @@ class MlFramework(str, Enum):
     HUGGINGFACE = "HF"
     MLX = "MLX"
     SKLEARN = "sklearn"
+    FLWRTUNE = "flwrtune"
+
+
+class LLMChallengeName(str, Enum):
```

Comment:
nit: in the `flower` repo, with all-caps abbreviations, we usually only put the first letter in caps. for example, we'd use `HttpRequest`, not `HTTPRequest`.

```suggestion
class LlmChallengeName(str, Enum):
```